test/api/pull/index.test.ts

```diff
@@ -47,6 +89,8 @@ describe("Pull API", () => {
         },
       });
     });
+
+    pullService = new PullService(sigRepository, sigMemberRepository);
```

Comment:
Why we need this?

Reply:
Because the `pullService ` determines whether to return collaborators as commiters and reviewers or not. Do you mean using the `mockPullService` return the collaborators list directly instead.

Reply:
Yes, I think we can just mock the service. Because this a API test.

Reply:
Good idea.